fuel-tx/src/transaction/checkable.rs

```diff
@@ -157,12 +157,17 @@ impl Output {
     }
 }
 
-/// Means that the transaction can be validated.
+/// Contains logic for stateless validations that don't result in any reusable metadata such
+/// as spendable input balances or remaining gas.
 pub trait Checkable {
```

Comment:
Maybe better to rename the trait itself into `BasicallyCheckable` or `SupportBasicChecks`? In this case, we don't need to rename `check` and only update the comment that it runs all basic checks=)

Reply:
Good idea, updated the PR. What do you think of `FormatValidityChecks`, as it seems a little less ambiguous than basic.

Reply:
Yep, good naming

Reply:
Maybe we should even consider renaming usages of `basic` in checked transactions to `formatted`?

Reply:
Then we need to do it for all "Basic"=D
<img width="606" alt="image" src="https://user-images.githubusercontent.com/18346821/213060956-60608228-1a6f-495b-b3f5-54dc49aaeb13.png">